system-addon/data/content/tippytop/top_sites.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "title": "amazon",
+    "urls": ["https://www.amazon.com/", "https://www.amazon.ca/", "https://www.amazon.de/", "https://www.amazon.co.uk/", "https://www.amazon.fr/"],
+    "image_url": "amazon-com@2x.png"
+  },
+  {
+    "title": "facebook",
+    "url": "https://www.facebook.com/",
+    "image_url": "facebook-com@2x.png"
```

Comment:
@k88hudson Not sure how long term this change would be, but one quick hack to get geo/country code into content would be to set it as a `getValue` pref:
https://github.com/mozilla/activity-stream/blob/007d0c1340cf3b1202533037ab8ef276fe947cdf/system-addon/lib/ActivityStream.jsm#L44
```js
["geo", {
  getValue: ({geo}) => geo
}]
```

And then content can access it via Prefs from the store.

Although, don't you actually want the geo from `TopSitesFeed` or `TippyTopProvider`? Those could just directly access `Services.prefs.getStringPref("browser.search.region")`. But if we do want geo everywhere, it should probably go in `store.App` alongside `locale` ?